docker/client.go

```diff
@@ -40,6 +44,35 @@ func NewClient() Client {
 	return &dockerClient{cli}
 }
 
+func (d *dockerClient) GetContainerById(containerId string) (Container, error) {
+    containerListOptions := types.ContainerListOptions{
+        Filters: filters.NewArgs(
+             filters.Arg("id", containerId),
+         ),
+    }
+    list, err := d.cli.ContainerList(context.Background(), containerListOptions)
```

Comment:
Would it make more sense to call `ListContainers()` instead and filter by id? That way the struct `Conatiner{}` doesn't have to be created again.

Reply:
I was thinking about this on my way to work. Since you are already using filters. Would it make more sense to just send the name pattern to docker instead and filter within `ListContainers`?

Reply:
When I did it this morning, thought it looked messier to have the additional logic in the `streamLogs` function, but you're prolly right.